js/map.js

```diff
@@ -285,36 +296,44 @@ map.on('load', () => {
   showLayer(selectedGeography, selectedCategory, selectedStatus)
   updateLegend(selectedGeography, selectedCategory, selectedStatus)
 
+  // calls the toggleActive function on the initial states
```

Comment:
we still need to handle the case when the map is loaded with different values set. each time a button is clicked it updates the URL with query parameters. we need to update the code here to account for selecting the right button